homeassistant/components/bosch_shc/strings.json

```diff
@@ -0,0 +1,37 @@
+{
+  "title": "Bosch SHC",
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set up your Bosch Smart Home Controller to allow monitoring and control with Home Assistant.",
+        "title": "SHC authentication parameters",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]"
+        }
+      },
+      "credentials": {
+        "data": {
+          "password": "Password of the Smart Home Controller"
+        }
+      },
+      "confirm_discovery": {
+        "description": "Please press the Bosch Smart Home Controller's front-side button until LED starts flashing.\nReady to continue to set up {model} @ {host} with Home Assistant?"
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "The bosch_shc integration needs to re-authenticate your account"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "pairing_failed": "Pairing failed... Please press the Bosch Smart Home Controller button until LED starts blinking and check your password.",
```

Comment:
Please don't use ellipsis. We should be clear and concise in the error message. Use a period or semi-colon.